app/install/macos.md

```diff
@@ -23,23 +23,42 @@ breadcrumbs:
     $ brew tap kong/kong
     $ brew install kong
     ```
+2. **(DB-less only) Write declarative configuration file**
```

Comment:
This could be merged with the section below as `Prepare your configuration`, with subsections for "Without a database", "Postgres" and "Cassandra".